datasetIngestor/checkMetadata.go

```diff
@@ -18,7 +18,10 @@ import (
 const DUMMY_TIME = "2300-01-01T11:11:11.000Z"
 const DUMMY_OWNER = "x12345"
 
+// getHost is a function that attempts to retrieve and return the fully qualified domain name (FQDN) of the current host.
+// If it encounters any error during the process, it gracefully falls back to returning the simple hostname or "unknown".
 func getHost() string {
```

Comment:
is this getHost ever used? if not, maybe we can remove in the next pr

Reply:
It is used. See line 138

Reply:
my bad, sorry!